MapboxNavigation/Constants.swift

```diff
@@ -52,6 +52,12 @@ public var RouteControllerManeuverZoneRadius: CLLocationDistance = 40
 
 
 /*
+ Maximum number of seconds the user can travel away from the start of the route before rerouting occurs
+*/
+public var MaxSecondsSpentTravelingAwayFromStartOfRoute: TimeInterval = 3
```

Comment:
We don’t consider the start of the route at all anymore, do we? https://github.com/mapbox/MapboxNavigation.swift/pull/84#discussion_r107560286